README.md

```diff
@@ -53,6 +53,7 @@ The following 38 microcontroller boards are currently supported:
 * [Adafruit Feather nRF52840 Express](https://www.adafruit.com/product/4062)
 * [Adafruit ItsyBitsy M0](https://www.adafruit.com/product/3727)
 * [Adafruit ItsyBitsy M4](https://www.adafruit.com/product/3800)
+* [Adafruit ItsyBitsy nrf52840](https://www.adafruit.com/product/4481)
```

Comment:
I think the same notation as the Adafruit Feather nRF52840 Express should be used. before : nrf52840

before : nrf52840
after : nRF52840

Reply:
Updated.